.github/scripts/commit/assert_commit_message.py

```diff
@@ -15,9 +15,12 @@ def assert_commit_message(branch_name):
     (True, "#100 Configure kotest"),
     (True, "#100 Configure-kotest"),
     (True, "#100 Configure-kotest"),
-    (False, "100 Configure"),
+    (True, "100 Configure"),
```

Comment:
Should we allow it? It will be hard to predict if someone put a standalone number intentionally, not connected with any issue, or just overlooked a `#` sign

Reply:
Good point, fixed